docs/language-server/protocol-language-server.md

```diff
@@ -4368,6 +4368,8 @@ Sent from client to the server to receive the autocomplete suggestion.
   returnType?: string;
   // Filter by the suggestion types
   tags?: [SuggestionEntryType];
+  // Filter by `static` attribute of the suggestion
+  isStatic?: Boolean;
```

Comment:
What is the default value when the attribute is missing?

Reply:
the TypeScript `?` notation in the type definition, i.e.
```ts
{
    a?: Type;
}
```
means that the `a` attribute is optional and may be `null`